ext/js/display/display.js

```diff
@@ -1417,15 +1416,17 @@ export class Display extends EventDispatcher {
         safePerformance.mark('display:contentUpdate:start');
         this._triggerContentUpdateStart();
 
-        for (let i = 0, ii = dictionaryEntries.length; i < ii; ++i) {
+        let i = 0;
+        for (const dictionaryEntry of dictionaryEntries) {
```

Comment:
Using a `for of` loop while still tracking `i` feels like a bad idea. I don't see a reason to not keep the loop style the same. Could use this to fix up the funky looking definition with this if you dont like `let i = 0, ii = dictionaryEntries.length`:
```js
let dictionaryEntriesLength = dictionaryEntries.length;
for (let i = 0; i < dictionaryEntriesLength; ++i) {
```